hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/HttpRequestFactory.java

```diff
@@ -52,6 +53,11 @@ Request buildRequest(Message message, URI uri) {
                 .timeout(timeout, TimeUnit.MILLISECONDS)
                 .content(new BytesContentProvider(message.getData()));
 
+        if (message.hasSubscriptionIdentityHeaders()) {
+            request.header(TOPIC_NAME.getName(), message.getTopic());
```

Comment:
Do we need to different headers ? Maybe it would be enough if you use SUBSCRIPTION_NAME as fully qualified name ?

Reply:
I don't know the requirements, but if there is no any specific we can think about it ;)